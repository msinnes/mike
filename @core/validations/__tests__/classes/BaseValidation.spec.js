const BaseValidation = require('../../classes/BaseValidation');

describe('BaseValidation', () => {
  it('should be an abstract class', () => {
    expect(() => {
      new BaseValidation('string');
    }).toThrowErrorMatchingSnapshot();

    expect(() => {
      BaseValidation('string');
    }).toThrowErrorMatchingSnapshot();
  });
});