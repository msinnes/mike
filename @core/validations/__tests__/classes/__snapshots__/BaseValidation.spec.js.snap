// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseValidation should be an abstract class 1`] = `"ClassError: Abstract classes cannot be instantiated"`;

exports[`BaseValidation should be an abstract class 2`] = `"ClassError: Cannot call a class as a function"`;
